modules/MIPS/generate_MIPS_xml.php

```diff
@@ -280,21 +225,14 @@ function get_TIN() {
 		fwrite($myFileHandle,  "          </value>\n");
 		fwrite($myFileHandle,  "      </measurement>\n");
         //end <measurement>
-
-
 	}	// End loop.  LOOP LOOP LOOP LOOP
-
-
 		fwrite($myFileHandle,  "   </measurementSet>\n");
 		//fwrite($myFileHandle,  " </submission>\n"); no submission element required.
 		fwrite($myFileHandle,  "</data>\n");
 		fclose($myFileHandle);
-
-
-
 } else {	// End if(!empty($report_id))
 	echo ("ERROR!  No report_id specified!\n");
 }
-
-
 ?>
+</body>
+</html>
```

Comment:
Need a newline at the end of all files.
NOTE:  This file is due for an update (code exists).  I will still implement the changes you have done here on committing the new version.

Reply:
@aethelwulffe @muarachmann Opening a new pull request, since I deleted the forked repo previously so it shows "wants to merge from unknown repository" and I am not able to make any further commits.

Reply:
Sorry about that.  Well, it is just the one file, so you should be able to pop up a new PR on a new branch (on your new fork I suppose) and pop it out with the changes.  We can merge it post haste after that... or I can just edit it here perhaps?  I will see.

Reply:
@aethelwulffe Yes, I made a new branch and have sent the pull request. Kindly review https://github.com/LibreHealthIO/lh-ehr/pull/1439.